pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3090,6 +3094,7 @@ private void initializeTopicSubscribeRateLimiterIfNeeded(Optional<TopicPolicies>
                 this.subscribeRateLimiter = Optional.of(new SubscribeRateLimiter(this));
```

Comment:
Similarly, is there a reason that we're not using `subscribeRateLimiter.onSubscribeRateUpdate(subscribeRate)` here?

Reply:
Topic-level  `subscribeRate` will update using `org.apache.pulsar.broker.service.AbstractTopic#updateTopicPolicy`.  And in the constructor method of `SubscribeRateLimiter` will get `subscribeRate` using `topic.getSubscribeRate()`. @michaeljmarshall